ci_stability.sh

```diff
@@ -20,13 +20,24 @@ hosts_fixup() {
     echo "travis_fold:end:hosts_fixup"
 }
 
+install_chrome() {
+    channel=$1
+    deb_archive=google-chrome-${channel}_current_amd64.deb
+    wget https://dl.google.com/linux/direct/$deb_archive
+    sudo dpkg --install $deb_archive || true
+    sudo apt-get install --fix-broken
```

Comment:
Sure. I've also refactored this with an `if` statement to make the relationship between the commands more clear.